profiling/Cargo.toml

```diff
@@ -29,10 +29,10 @@ futures = { version = "0.3", default-features = false }
 futures-core = {version = "0.3.0", default-features = false}
 futures-util = {version = "0.3.0", default-features = false}
 hashbrown = { version = "0.14", default-features = false, features = ["allocator-api2"] }
-http = "0.2"
-http-body = "0.4"
-hyper = {version = "0.14", features = ["client", "backports", "deprecated"], default-features = false}
-hyper-multipart-rfc7578 = "0.7.0"
+http = "1.0"
+hyper = { version = "1.6", features = ["http1", "client"] }
+http-body-util = "0.1"
+hyper-multipart-rfc7578 = "0.9.0"
```

Comment:
Erm... [this on on 0.8.0](https://github.com/ferristseng/rust-multipart-rfc7578/blob/8e825fae2672ba71a9b65e2ce4cd014ef221271a/hyper/Cargo.toml#L9)? Where did 0.9.0 come from?!

Reply:
It's pushed on on crates io  https://crates.io/crates/hyper-multipart-rfc7578 
From the package [metadata](https://docs.rs/crate/hyper-multipart-rfc7578/0.9.0/source/.cargo_vcs_info.json) this was pushed from this commit https://github.com/ferristseng/rust-multipart-rfc7578/tree/750c6d034c6460fedbb5423031bb286ff74cb9b5

Not sure why this isn't merged in main?

Reply:
The diff between main and the pushed branch is minimal. https://github.com/ferristseng/rust-multipart-rfc7578/compare/main...750c6d034c6460fedbb5423031bb286ff74cb9b5

But it includes a maintainer call, so this is probably unmaintained now.